platform/android/src/storage/default_file_source_peer.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include <jni/jni.hpp>
+
+#include <memory>
+
+namespace mbgl {
+
+class DefaultFileSource;
+
+namespace android {
+
+class DefaultFileSourcePeer {
```

Comment:
I decided to use it because we'll get conflicts with the regular `DefaultFileSource` otherwise. In the style code, we're using absolutely qualified identifiers `mbgl::style::SymbolLayer` in those places, which works, but is just another source of potential conflicts.